.github/workflows/docs.yml

```diff
@@ -0,0 +1,46 @@
+name: Docs
+
+on:
+# Run this workflow only for commits to master
+  # push:
+    # branches: [master]
+# Run this workflow only when a pull request is closed
+  pull_request:
+    types: [closed]
+
+jobs:
+  build:
+    name: Build docs
+    # Only proceed if the closed PR was also merged
+    # (see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-when-a-pull-request-merges)
+    if_merged:
+        if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks out a copy of your repository on the ubuntu-latest machine
```

Comment:
From [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-when-a-pull-request-merges) I thought that everything to be run needed to be under the if merged statement. What do you think?